snappass/templates/expired.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+
+{% block content %}
+<div class="container">
+  <section>
+    <div class="page-header"><h1>Secret not found, used or expired</h1></div>
+    <p class="lead">The requested URL was not found on the server. This could be because this URL never contained a secret, or because it expired or revealed earlier.</p>
```

Comment:
Small grammar improvement here:

```suggestion
    <p class="lead">The requested URL was not found on the server. This could be because this URL never contained a secret, or because it expired or was revealed earlier.</p>
```